content/dev/plugins/encapsulated_plugins/plugin_language_files.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Plugin Installer Language Files
+description:  
+weight: 20
+layout: docs
+---
+
+## Plugin Installer Language Files
+
+While most language constants for the plugin installer are managed internally there will be cases where a plugin
+needs to define some of it's own language defines.
+
+For example, if the plugin needs to test some pre-requisites befor installing, the error messages when those
```

Comment:
Causally reading for my own understanding and my usual pickiness has reared its head: Missing letter `e` in the word `before`

Reply:
Corrected. Thanks.